nx.json

```diff
@@ -114,7 +114,7 @@
     },
     "codegen/backend-client": {
       "dependsOn": ["^codegen/backend-client", "^codegen/backend-schema"],
-      "inputs": ["schema-files", "^schema-files"]
+      "inputs": ["schema-files", "^schema-outputs"]
```

Comment:
I was reviewing Deloittes car recycling PR #12879 and there we are creating backend client for GQL api, not rest. Don't we need to add GQL files here as inputs for backend-client to resolve Nx cache issues that could happen?

Reply:
Good point. Added 👍